tools/benchmark/utils/constants.py

```diff
@@ -28,9 +28,11 @@
 XML_EXTENSION = '.xml'
 BIN_EXTENSION = '.bin'
 BLOB_EXTENSION = '.blob'
+ONNX_EXTENSION = '.onnx'
```

Comment:
`prototxt` is also used to store ONNX models (see https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/inference_engine/ie_network_reader.cpp#L119)

CC @ilyachur

Reply:
It's also need to update python API, because there is a check only for ".onnx" model format.

Reply:
Why do you need update the Python API? Do you have some checks on the Python level?

Reply:
Maybe better remove this check at all from python benchmark_app and rely on API always? Anyway API should be fixed as well to enable prototxt

Reply:
> Anyway API should be fixed as well to enable prototxt

Which API? C++ accepts prototxt

Reply:
@ilya-lavrenov https://github.com/openvinotoolkit/openvino/blob/45d1b4eb19d1f4eaaef72f7c94af0a30ed062e51/inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx#L280

Reply:
It is also not correct! Because IR can be loaded without weight if model doesn't require weights.
maybe we should remove all redundant checks from python API all incorrect cases should be covered by C++.

Reply:
Discussed offline with @ilya-lavrenov: keep checks in API as is, remove check for extension from benchmark_app